Library/Homebrew/cmd/outdated.rb

```diff
@@ -97,8 +98,10 @@ def print_outdated(formulae_or_casks, args:)
           elsif f.head? && outdated_kegs.any? { |k| k.version.to_s == f.pkg_version.to_s }
             # There is a newer HEAD but the version number has not changed.
             "latest HEAD"
-          else
+          elsif f.tap.present?
             f.pkg_version.to_s
+          else
```

Comment:
```suggestion
          elsif ENV["HOMEBREW_JSON_CORE"].present?
```

maybe?

Reply:
Good call, forgot about the variable for these 👍